CHANGELOG.md

```diff
@@ -28,12 +28,13 @@
 - Support PodDisruptionBudgets ([#330]).
 - Added support for versions 2.6.3, 2.7.2 ([#334]).
 - Support graceful shutdown ([#343]).
+- Use new label builders ([#364]).
 
 ### Changed
 
 - [BREAKING] Consolidated `spec.clusterConfig.authenticationConfig` to `spec.clusterConfig.authentication` which now takes a vector of AuthenticationClass references  ([#303]).
 - `vector` `0.26.0` -> `0.33.0` ([#308], [#334]).
-- `operator-rs` `0.44.0` -> `0.55.0` ([#308], [#330], [#334]).
+- `operator-rs` `0.44.0` -> `0.59.0` ([#308], [#330], [#334], [#364]).
```

Comment:
Just FYI, we decided to not track operator-rs upgrades any further. So you don't need to add it to `[Unreleased]`

Reply:
removed in 3b6e53242c1d8c5fcfb8fc726a97f582d441f240

Reply:
also 3f32d14c389ac17acbe53716fb6bd3ab3d7cb767